api/controllers/kubernetes/install/controller_test.go

```diff
@@ -90,6 +90,8 @@ func TestGetInstallationConfig(t *testing.T) {
 		},
 	}
 
+	t.Setenv("HELM_BINARY_PATH", "helm") // use the helm binary in PATH
```

Comment:
Does this mean i need helm install locally to run unit tests?

Reply:
yes, any other ideas? helm binary is currently a requirement for the dev env anyways.

Reply:
if you were to instantiate only one client as suggested you would not have to set the env var